### ISSUE TITLE
Noticeable stutters while scrolling and using keyboard

### ISSUE THREAD

**Issue by ahmark-new** [TJYSunset/Phocid#16]
There are noticeable stutters while scrolling through songs and more noticeable while Keyboard closing.

https://github.com/user-attachments/assets/8d42d37c-5e39-4e7a-8203-611b9732f274

**Comment by TJYSunset** [comment]
Fixed locally. Will publish a new version tomorrow.

**Comment by TJYSunset** [comment]
Does this build solve your issue?
https://github.com/TJYSunset/Phocid/releases/tag/v20241208

The reasons for UI performance problems are complicated. I've tried eliminating the most obvious ones, but the player screen still lags like hell when the queue is 5000 tracks big (probably not my fault).

**Comment by ahmark-new** [comment]
Yep, It's a lot better now. Everything's working like butter now except the now playing interface. It's not exactly stuttering, In simple terms, Whole app works on 120 fps (smooth) but as soon as now playing Interface is opened or closed, it feels like animations(opening and closing) are working on 30-40 fps. But all the other components are now smooth.

**Comment by TJYSunset** [comment]
Low fps in now playing screen mostly fixed in b575cbf04c1588943b10d265708ca18e74d91f0e. Still stutters for large queues though.

**Comment by ahmark-new** [comment]
How about adding a limit to queue that user can set according to their preference.
For instance, like in RiMusic. 
![Screenshot_2024-12-09-13-20-08-482_it fast4x rimusic](https://github.com/user-attachments/assets/a2c5ce80-491d-4570-9534-9907acf8beb8)
